mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -186,6 +186,10 @@ The `config` block supports:
 * `maintenance_window` -
   (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
   The configuration settings for Cloud Composer maintenance window.
+  
+* `workloads_config` -
```

Comment:
This field is not beta correct?

Reply:
can you expand `workloads_config` subfields inline with the other block types

ie: The `workloads_config` block supports: